aries-site/src/pages/foundation/voice-and-tone.mdx

```diff
@@ -1 +1,361 @@
-More content coming soon.
+## Messaging Voice, Tone, and Content
+
+Depending on where you look, you will find different definitions for tone and voice.  By reading through many of these different definitions, a picture emerges:
+
+| **Voice**		                  | **Tone**                                    |
+| ----------------------------- | ------------------------------------------- |
+| Who they hear		              | What they hear                              |
+| Personality		                | Attitude                                    |
+| How you are doing the writing	| Who they hear                               |
+| Style		                      | How your message comes across               |
+| What makes us uniquely us		  | How we communicate to our specific audience |
+
+### Voice
+
+Voice is all about who the reader hears talking in our messaging; it has to do with the worldview of the author. Voice can be spoken, witty, friendly, formal, etc. We can be perceived as being intelligent, experienced, mature, accepting, objective, etc.
+
+A person's voice typically does not change once established. If our voice (HPE's voice) is inconsistent, people will notice. For example, as one notices the difference between writing academically when compared to sounding like a 5-year old. This kind of inconsistency can result in lack of trust.
+
+### Tone
+
+Tone has to do with what is conveyed or how it comes across. It can express seriousness, urgency, playfulness, objectivity, intimacy, condescension, etc.
+
+Unlike voice, tone can change based on the target audience and the subject matter. It is situational. For example, tone is very different when giving directions during an emergency vs ordering a pizza.
+
+### Point of View
+
+Point of view has to do with the perspective from which the message is written. First person perspective uses words like: I, me, us, we, our, etc. Second person uses any form of the word "you" which has the effect of addressing the reader. Third person directs the reader to things that are neither the writer or the reader.
+
+Examples:
+
+- *We noticed the server is malfunctioning.* (1st person-ish)
+- *Your server is malfunctioning.* (2nd person-ish)
+- *The server is malfunctioning.* (3rd person - no focus on the writer or the reader)
+
+### Our Voice and Tone
```

Comment:
I think leading this page with ground rules on what voice, tone, and point of view are is good. However, I'm wondering if there is any way to make this section a little more apparent as a focus area for people coming into the page.